[BUG] `make` from Capability Provider tutorial fails

### Affected project(s)

- [X] documentation
- [X] examples
- [ ] wasmCloud host
- [ ] wasmCloud CLI (wash)
- [ ] wasmCloud dashboard UI (washboard)
- [x] capability providers
- [ ] provider bindgen
- [ ] control interface client
- [ ] other / not sure

### Describe the bug

I'm trying to implement a Capability Provider from the official WasmCloud [tutorial](https://wasmcloud.com/docs/developer/interfaces/creating-an-interface) but `make` command for building Rust crate is failing with the following error:
```
make
cd rust && cargo build
   Compiling wasmcloud-examples-payments v0.1.0 (/Users/<username>/dev/wasm_hello_world/payments/rust)
error: failed to run custom build command for `wasmcloud-examples-payments v0.1.0 (/Users/<username>/dev/wasm_hello_world/payments/rust)`

Caused by:
  process didn't exit successfully: `/Users/<username>/dev/wasm_hello_world/payments/rust/target/debug/build/wasmcloud-examples-payments-419c609ccc7089a6/build-script-build` (exit status: 1)
  --- stdout
  cargo:rerun-if-changed=/Users/<username>/dev/wasm_hello_world/payments/codegen.toml
  cargo:rerun-if-changed=/Users/<username>/dev/wasm_hello_world/payments/./payments.smithy
  cargo:rerun-if-changed=/Users/<username>/Library/Caches/smithy/wasmcloud.github.io/wasmcloud-core.f2facc62766ec8c4.smithy
  cargo:rerun-if-changed=/Users/<username>/Library/Caches/smithy/wasmcloud.github.io/wasmcloud-model.6e8584f5be28601b.smithy

  --- stderr
  Error: Formatter("failed to start: No such file or directory (os error 2)")
make: *** [all] Error 101

```

### Steps to reproduce

1. Follow the official tutorial for creating an interface for a Capability Provider
2. After replacing `payments.smithy` with the example provided on Github run `make`

### Expected behavior

Rust cargo should be build successfully.be built

### Environment

- OS: MacOS
- Shell: zsh
- wash CLI Version: 0.25.0


### Screenshots / Logs / Additional context

_No response_

--- a/docs/developer/interfaces/creating-an-interface.md
+++ b/docs/developer/interfaces/creating-an-interface.md
@@ -246,4 +246,11 @@ These check the file against a predefined set of rules. The rules will be added 
 
 You can type `make` to generate the code and build the Rust library.
 
+:::info[Note]
+You will need [`rustfmt`](https://github.com/rust-lang/rustfmt) installed to successfully run `make`. To install with `rustup`:  
+```shell
+rustup component add rustfmt
+```
+:::
+
 Now that we have a Rust crate that we can use from both our actor and our capability provider, let's move on to the next step: [_Creating a Capability Provider in Rust_](/docs/developer/providers/rust.md).

